### ISSUE TITLE
[Mentorship Diary] Dan and Bamieh

### ISSUE THREAD

**Issue by Bamieh** [nodejs/mentorship#79]
# [Mentorship Diary] Dan and Bamieh

_(Mentorship Round 1)_ @dshaw will be mentoring [@bamieh](https://github.com/bamieh) on commcomm.

**Comment by Bamieh** [comment]
# Kickoff meeting (15 July 2018)
Due to some technical issues we failed to record the meeting.

## Scope Introduction
Dan will be mentoring in the CommComm Scope, mainly around taking off and managing commcomm initiatives.

> The Community Committee works to empower people in every part of the project. By making the Node.js project more diverse and improving the environment for inclusivity, we attract a wider range of views, voices, and opinions, which in turn helps us ship better software.


## Goals & Expectations
- Get Dan's expertise in managing Open source project/initiatives
- Get feedback on the decisions I make in both commcomm and mentorship
- Dan wants to make sure that I have more confidence making decisions in the project.

## Availability
Dan does not prefer slack to setup the meetings. We'll be doing 30 minutes meetings each week, the meeting time and date will be _flexible_. Meetings will be held on Zoom.

## Privacy & CoC
We are both comfortable recording sessions and will record sessions and link to them on github.

**Comment by Bamieh** [comment]
**Milestone 1**

As a fist milestone, I want to be able to lead meetings better.
- Taking notes and leading meeting is not compatible,
- Establish need of audience
- Team building: establishing a team that communicates and reacts, and know why they are there.
- Digest what the community wants, listen and engage, label things for discussion and prepare for it before the meeting.
- For the person leading the meeting, the real action happens at the beginning and at the end. The beginning of the meeting is very important to get everyone's attention and ensure a successful meeting.
- Guest discussion items must be on the top of the agenda, this way the guests are not forced to wait long for their item to come up, and this encourages them to join more meetings in the future.
- Do a round of introductions to get everyone talking, introductions lessen the tension and fill time.
- When brainstorming, ask the attendance that you want to go in circle asking everyone their input, and tell them that they are free to skip if they want to.

**Comment by Bamieh** [comment]
**2nd Meeting (31 July 2018)**

Recording: _To be uploaded soon_

I watched how Dan and Tierney lead meetings, and I took notes about how to go about starting a meeting. To make sure that the milestone is covered, Dan and I watch a little bit of the last mentorship meeting so Dan can provide feedback on how I lead the meeting.

- The beginning of the meeting is very important to get everyone's attention and ensure a successful meeting.
- Guest discussion items must be on the top of the agenda, this way the guests are not forced to wait long for their item to come up, and this encourages them to join more meetings in the future.
- Do a round of introductions to get everyone talking, introductions lessen the tension and fill time.
- When brainstorming during a meeting online, ask the attendance that you want to go in circle asking everyone their input, and tell them that they are free to skip if they want to.


We also discussed how we can get the mentorship initiative more visible to Node.js audience during the collaborator summit.

**Comment by Bamieh** [comment]
**4th Meeting (11 Sept 2018)**

_Recording: Not recorded._

We had a 20 minutes meeting.

- We discussed mentorship and created a small agenda on how to present mentorship in the Vancouver event for Node+JS.
- We discussed a little about how to structure mentorship, Dan shared a little bit of his experience with scaling in OSS.
- Discussed ways to recruiting more people for mentorship, reach out to current mentees
- I will write a prototype for the slides to be presented on mentorship and share them with dan and the mentorship team to iterate over and finalize.
